src/configs/FirebaseConfig.js

```diff
@@ -35,6 +35,7 @@ const FirebaseConfig = (props) => {
     if (dynamicLink?.url?.includes('postPrivateId=')) return handlePrivatePost(dynamicLink);
     if (dynamicLink?.url?.includes('communityName')) return handleCommunityPage(dynamicLink);
     if (dynamicLink?.url?.includes('users?username=')) return getUserProfile(dynamicLink?.url);
+    if (dynamicLink?.url?.includes('profile/')) return getUserProfileV2(dynamicLink?.url);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code checks if the URL includes 'profile/' and then calls `getUserProfileV2` function. However, it doesn't validate if the URL also contains the 'userId' parameter which is necessary for the `getUserProfileV2` function to work correctly. This could lead to unexpected behavior if a URL with 'profile/' but without 'userId' is passed.
```diff
-    if (dynamicLink?.url?.includes('profile/')) return getUserProfileV2(dynamicLink?.url);
+    if (dynamicLink?.url?.includes('profile/') && dynamicLink?.url?.includes('?userId=')) return getUserProfileV2(dynamicLink?.url);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->